rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -2579,8 +2604,19 @@ void opcode_00d7(const ScriptArguments& args, const ScriptLabel arg1) {
 	opcode 00d8
 */
 void opcode_00d8(const ScriptArguments& args) {
-	RW_UNIMPLEMENTED_OPCODE(0x00d8);
-	RW_UNUSED(args);
+	/// @todo verify behaviour
+	for( auto oid : args.getState()->missionObjects )
+	{
+		auto obj = args.getWorld()->vehiclePool.find(oid);
+		if( obj )
+		{
+			args.getWorld()->destroyObjectQueued(obj);
+		}
+	}
+	
+	args.getState()->missionObjects.clear();
+	
+	*args.getState()->scriptOnMissionFlag = 0;
```

Comment:
I'm not sure if you also intent to fix formatting with this, but this has a lot of useless line breaks which aren't done in the code above.